pallets/account/Cargo.toml

```diff
@@ -0,0 +1,76 @@
+[package]
+name = "pallet-account"
```

Comment:
About the pallet naming - to me this is again confusing 🙂 

Inside the code **address**, **account** and **accountId** are used all over the place.
Pallet name is **account**.
What does that imply?

Maybe something like `pallet-unified-accounts` would be better?
And definitely getting the nomenclature in order in the code.

Reply:
I'm bad with names apparently.
I'll revamp the naming to be more consistent and precise. I initially wanted the pallet to be very generic, but decided not to later on, that lead to a lot of inconsistent naming.

`pallet-unified-accounts` sound good